client/Bootstrap/index.ts

```diff
@@ -7,7 +11,7 @@ import 'Styling';
 import image from 'Images/logo.png';
 
 window.addEventListener('load', () => {
-  const root = document.querySelector('[id="root"]');
+  const root = document.querySelector('[id="root"]') as Element;
```

Comment:
This satsifies the typing, but wouldn't `document.querySelector('[id="root"]') || document.createElement('div');` be more robust (as a div will be appended to the document body if root is not found)?

Reply:
It's in the index.html so not sure how it wouldn't be found.  I'll add this though.